micrometer-spring-legacy/src/samples/java/io/micrometer/spring/samples/library/ProposalLibrary.java

```diff
@@ -0,0 +1,17 @@
+package io.micrometer.spring.samples.library;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Meters;
+
+public class ProposalLibrary {
+
+    //@Inject no available on classpath
```

Comment:
Sorry the comment was for why I hadn't included it in the code here, since `javax.inject` wasn't in the project right now.

As a library developer I wouldn't want to include it, unless it works as an optional dep so i can leave it as an annotation even if the downstream user isn't going to use it